pkg/ReadMe.md

```diff
@@ -0,0 +1,84 @@
+# How to make a release
+
+## Versioning
+
+Update libyaml version in:
+* announcement.msg
+* Changes
+* CMakeLists.txt
+  * `YAML_VERSION_MAJOR`, `YAML_VERSION_MINOR`, `YAML_VERSION_PATCH`
+* .appveyor.yml
+* configure.ac
+  * `YAML_MAJOR`, `YAML_MINOR`, `YAML_PATCH`, `YAML_RELEASE`, `YAML_CURRENT`, `YAML_REVISION`
+
+Commit and push everything to release/0.x.y
+
+## Test and Create Release archives
+
+### GitHub Actions Automation
+
+The github workflow
+
+    .github/workflows/dist.yaml
+
+will do this automatically for you.
+
+#### .github/workflows/dist.yaml
+
+Will create release archives (tar.gz and zip)
+
+### Manually
+
+Make sure you have a clean git repository (no changed files).
+The following process will clone your current git directory.
+
+This will need the docker image `yamlio/libyaml-dev`.
+You can either pull it, or create it yourself:
+
+    make docker-build
+
+### Create dist archives
+
+Run:
+
+    make docker-dist
+
+It will run `make dist` in the container to create a tarball written to
+pkg/docker/output.
+It will also create a zipfile.
+
+## Update master
+
+    git merge release/0.x.y
+    git tag -a 0.x.y
+    # <Editor opens>
+    # Paste the corresponding entry from the Changes file
+    # Look at an earlier release for how it should look like:
+    #   git show 0.2.3
+    git push origin master 0.x.y
+
+## Create a GitHub release
+
+Go to "Releases" and click on "Draft a new release".
+
+Fill in the tag you just created in the previous step.
+
+Fill in the release title: v0.x.y
+
+Paste the changelog into the description field.
+
+Upload the tar.gz and .zip file.
+
+You can "Save draft" and publish later, or directly click on "Publish release".
+
+## Update pyyaml.org
```

Comment:
Do we still want this section?

Reply:
I'm not sure why you think it wouldn't be needed anymore?

Reply:
> PyYAML is just a consumer of libyaml. They are separate projects that happen to have the same owner. Documenting stuff about PyYAML in liibyaml doesn't make sense to me.

libyaml is documented on pyyaml.org.
https://pyyaml.org/ lists all pyyaml and libyaml releases.
https://pyyaml.org/wiki/LibYAML documentes libyaml and where to find the last release.
All libyaml release tarballs can be downloaded from https://pyyaml.org/download/libyaml/

Reply:
Some of that makes sense because libyaml is a dependency of pyyaml, but not the other way around.

I don't really want specific mention of a project that uses libyaml. Especially not instructions on how that project should handle libyaml releases.

However it is important to notify major projects using libyaml of new releases.

How about we change this sections to
```
## Notify major projects that depend on libyaml

Many projects (like PyYAML) depend on libyaml and should be notified when new versions are released.
See this [this wiki page](https://github.com/yaml/libyaml/wiki/Projects-that-depend-on-libyaml) for more information.
```

Reply:
I don't understand. This is not about notifying other projects, this is about updating a website that hosts *both* docs and releases for pyyaml and libyaml.
If you don't want libyaml news and tarballs to be on this site any more, you should mention this there instead of just not updating it anymore.

I now added a [ReadMe](https://github.com/yaml/pyyaml.org/blob/master/ReadMe.md) to pyyaml.org and simply linked to it from here.

This is my last suggestion.

Reply:
OK. Thanks. I didn't fully understand.
I like what you have now.